### ISSUE TITLE
html lang="en" when locale: non-en

### ISSUE THREAD

**Issue by sergeykorobeynikov** [retypeapp/retype#574]
retype.yml:
```yml
locale: ru
```

The source code of any html page that has been generated:
```html
<!DOCTYPE html>
<html lang="en" class="h-full">
```
This is bug?

P.S. Retype 3.0.3

**Comment by geoffreymcgill** [comment]
Confirmed, defect. 

I'm not sure how we missed correctly setting the `lang` attribute, but we did. 

This should be an easy fix and we will get this into the next release. There will be no changes required on your side, other than upgrading to that next release.

Thanks for letting us know about this issue.

**Comment by sergeykorobeynikov** [comment]
OK. This may somehow affect the work of search services.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the upcoming Retype v3.1 release.

```html
<html lang="ru" class="h-full">
```

Thanks for reporting the defect.
